packages/record-data/addon/-private/normalize-link.ts

```diff
@@ -1,19 +1,18 @@
-/**
-  @module @ember-data/store
-*/
-
 /*
   This method normalizes a link to an "links object". If the passed link is
   already an object it's returned without any modifications.
 
   See http://jsonapi.org/format/#document-links for more information.
 
   @method _normalizeLink
-  @private
+  @internal
   @param {String} link
   @return {Object|null}
 */
-export default function _normalizeLink(link) {
+type Link = import('@ember-data/store/-private/ts-interfaces/ember-data-json-api').Link;
```

Comment:
Seems better to import from `-private`, no?

Reply:
Same reasoning as above, we would have to re-export all the types which seems bad